webapp/security/views.py

```diff
@@ -258,3 +259,22 @@ def api_create_notice():
     db_session.commit()
 
     return flask.jsonify({"message": "Notice created"}), 201
+
+
+# CVE views
+# ===
+
+
+def cve(cve_id):
+    print(cve_id)
+    match_cves = re.compile(r"cve-\d{4}-\d{4,7}")
```

Comment:
I think for performance you should do the regex compile outside the view.

Reply:
Taken outside